### ISSUE TITLE
config: `killProcessOnBan`

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#3081]

**Comment by smashah** [comment]
true by default

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.61.2](https://github.com/open-wa/wa-automate-nodejs/compare/4.61.1...4.61.2) (2023-03-28)

- Release 4.61.2 [`00dd728`](https://github.com/open-wa/wa-automate-nodejs/commit/00dd7285b44db57a59bca9aff32053e0d42981a5)
- 🚑 fix: 6.41.1 regression mitigated #3080 [`7790c4d`](https://github.com/open-wa/wa-automate-nodejs/commit/7790c4d1ab8f3cb3be80dd0f468fbf30236a8a5a)
- ✨ `config.killProcessOnBan` default `true` #3081 [`b586011`](https://github.com/open-wa/wa-automate-nodejs/commit/b586011b42001c3b5254315c589d36e6e1295500)
- 🥚 patches update [`ad23dd3`](https://github.com/open-wa/wa-automate-nodejs/commit/ad23dd35b77d7c03e97182acaa6f20ebc76bf76d)
- updated types-only package [`1479e4f`](https://github.com/open-wa/wa-automate-nodejs/commit/1479e4f6d452b6b0c5c8b9efd07d4c888d4c7335)
